README.md

```diff
@@ -1 +1,207 @@
+
+- At present, InstancePrincipal and OCI Config File (UserPrincipal) based Auth/AuthZ are supported for Fluentd to talk to OCI Logging Analytics APIs. 
+- We recommend to use InstancePrincipal based AuthZ for OKE and all clusters which are running on OCI VMs and that is the default auth type configured. 
+- Applying this file is not required when using InstancePrincipal based auth type.
+- You need to modify this file to fill out the values under config section with appropriate values.
+
+##### Commands Reference
+
+Apply the yaml files in the sequence of configmap-docker.yaml(or configmap-cri.yaml), secrets.yaml (not required for default auth type) and fluentd-daemonset.yaml.
+
+```
+$ kubectl apply -f configmap-docker.yaml 
+configmap/oci-la-fluentd-logs-configmap created
+
+$ kubectl apply -f secrets.yaml 
+secret/oci-la-credentials-secret created
+
+$ kubectl apply -f fluentd-daemonset.yaml 
+serviceaccount/oci-la-fluentd-serviceaccount created
+clusterrole.rbac.authorization.k8s.io/oci-la-fluentd-logs-clusterrole created
+clusterrolebinding.rbac.authorization.k8s.io/oci-la-fluentd-logs-clusterrolebinding created
+daemonset.apps/oci-la-fluentd-daemonset created
+```
+
+You may use the following command to restart DaemonSet upon any applying any modifications to configmap or secrets to reflect the changes into the Fluentd.
+
+```
+kubectl rollout restart daemonset oci-la-fluentd-daemonset -n=kubectl
+```
+
+#### To enable Kubernetes Objects collection
+
+Download all the yaml files from [this dir](/logan/kubernetes-resources/objects-collection/).
+These yaml files needs to be applied using kubectl to create the necessary resources that enables the Kuberetes Objects collection into Logging Analytics.
+
+##### configMap-objects.yaml
+
+- This file contains the necessary out of the box fluentd configuration to collect Kubernetes Objects.
+- Refer [this](https://docs.oracle.com/en/learn/oci_logging_analytics_fluentd/) to learn about each of the Logging Analytics Fluentd Output plugin configuration parameters.
+
+##### fluentd-deployment.yaml
+
+Refer [this](#fluentd-daemonsetyaml) section.
+
+##### secrets.yaml (Optional)
+
+Refer [this](#secretsyaml-optional) section.
+
+##### Commands Reference
+
+Apply the yaml files in the sequence of configmap-objects.yaml, secrets.yaml (not required for default auth type) and fluentd-deployment.yaml.
+
+```
+$ kubectl apply -f configmap-objects.yaml 
+configmap/oci-la-fluentd-objects-configmap configured
+
+$ kubectl apply -f fluentd-deployment.yaml 
+serviceaccount/oci-la-fluentd-serviceaccount unchanged
+clusterrole.rbac.authorization.k8s.io/oci-la-fluentd-objects-clusterrole created
+clusterrolebinding.rbac.authorization.k8s.io/oci-la-fluentd-objects-clusterrolebinding created
+deployment.apps/oci-la-fluentd-deployment created
+```
+
+You may use the following command to restart DaemonSet upon any applying any modifications to configmap or secrets to reflect the changes into the Fluentd.
```

Comment:
Use the following command to restart DaemonSet upon applying any modifications to configmap or secrets, to reflect the changes into the Fluentd.